utils/ibmsetup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -eux
+
+DETAILSFILE=$(mktemp)
+trap "rm -f $DETAILSFILE" EXIT
+
+dnf install -y openssh-clients openssh jq which python3-netaddr python3-dns iputils ansible which diffutils findutils jq
+
+BASEDIR=$(dirname $0)
+
+which ibmcloud || curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
+
+python3 $BASEDIR/ibmsetup.py > $DETAILSFILE
+cat $DETAILSFILE
```

Comment:
is this needed?

Reply:
Not needed, but I did intend on leaving it there so that the info is logged